### PR TITLE
W-10964385: Make ExtensionModelDiscoverer#discoverRuntimeExtensionModels static

### PR DESCRIPTION
**Pull request by jcgrethe**

### CODE REVIEW

**Comment by jcgrethe** [comment]
--validate

**Comment by jcgrethe** [comment]
--validate

**Comment by jcgrethe** [comment]
--validate

**Review by rbourbonnavarro — DISMISSED** [review]

File: modules/artifact-activation/src/main/java/org/mule/runtime/module/artifact/activation/internal/extension/discovery/DefaultExtensionModelDiscoverer.java [hunk 1]
```diff
@@ -57,7 +44,7 @@ public Set<ExtensionModel> discoverPluginsExtensionModels(ExtensionDiscoveryRequ
                    if (!dependencies.contains(MuleExtensionModelProvider.getExtensionModel())) {
                      dependencies = ImmutableSet.<ExtensionModel>builder()
                          .addAll(extensions)
-                         .addAll(discoverRuntimeExtensionModels())
+                         .addAll(ExtensionModelDiscoverer.discoverRuntimeExtensionModels())
```
Comment (elrodro83) [inline modules/artifact-activation/src/main/java/org/mule/runtime/module/artifact/activation/internal/extension/discovery/DefaultExtensionModelDiscoverer.java:47]: static import

**Review by elrodro83 — CHANGES_REQUESTED** [review]

**Review by elrodro83 — APPROVED** [review]

**Pull request merged**